File: Imetame.Documentacao.Domain/Models/ColaboradorModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Imetame.Documentacao.Domain.Models
{
	public class ColaboradorModel
	{
		public string Empresa { get; set; }
		public string NumCad { get; set; }
		public string NumCracha { get; set; }
		public string Status { get; set; }
		public string Nome { get; set; }
		public string Cpf { get; set; }
		public string FuncaoAtual { get; set; }
		public string FuncaoInicial { get; set; }
		public DateTime DataAdmissao { get; set; }
		public DateTime DataNascimento { get; set; }
		public bool SincronizadoDestra { get; set; } = false;
		public string Equipe { get; set; }
		public string Perfil { get; set; }
		public string Endereco { get; set; }
		public string Numero { get; set; }
		public string Bairro { get; set; }
		public string Cidade { get; set; }
		public string Cep { get; set; }
		public string Ddd { get; set; }
		public string Numtel { get; set; }
		public string Estado { get; set; }
		public string TempoEmpresaAnos { get; set; }
		public string TempoEmpresaAnosInt { get; set; }
		public int CountDocumento { get; set; }

		// DEFINIDO COMO INT TEMPORARIAMENTE ATÉ QUE TENHA AS RELAÇÕES DO ENUM DA DESTRA
		[NotMapped]
		public int StatusDestra { get; set; } = -2;

		[NotMapped]
		public bool IsAssociado { get; set; } = false;
	}
	public class ColaboradorProtheusModel
	{
		public string MATRICULA { get; set; }
		public string CRACHA { get; set; }
		public string NOME { get; set; }
		public string NASCIMENTO { get; set; }
		public string CPF { get; set; }
		public string RG { get; set; }
		public string DATA_ADIMISSAO { get; set; }
		public string CODIGO_FUNCAO { get; set; }
		public string NOME_FUNCAO { get; set; }
		public string CODIGO_EQUIPE { get; set; }
		public string NOME_EQUIPE { get; set; }
		public string CODIGO_DISCIPLINA { get; set; }
		public string NOME_DISCIPLINA { get; set; }
		public string PERFIL { get; set; }
		public string CODIGO_OS { get; set; }
		public string NOME_OS { get; set; }
	}
}

File: Imetame.Documentacao.WebSPA/src/app/models/DTO/ColaboradorModel.ts
import { BaseEntity } from "../BaseEntity";
import { ColaboradorxAtividade } from "../ColaboradorxAtividade";


export class ColaboradorModel extends BaseEntity {
    Empresa: string;
    NumCad: string;
    NumCracha: string;
    Status: string;
    Nome: string;
    Cpf: string;
    FuncaoAtual: string;
    FuncaoInicial: string;
    DataAdmissao: Date;
    DataNascimento: Date;
    Equipe: string;
    Perfil: string;
    Endereco: string;
    Numero: string;
    Bairro: string;
    Cidade: string;
    Cep: string;
    Ddd: string;
    Numtel: string;
    Estado: string;
    TempoEmpresaAnos: string;
    TempoEmpresaAnosInt: string;
    CountDocumento: number;

    ColaboradorxAtividade?: ColaboradorxAtividade[];
}

export class ColaboradorProtheusModel extends BaseEntity {
    MATRICULA: string;
    CRACHA: string;
    NOME: string;
    NASCIMENTO: string;
    CPF: string;
    RG: string;
    DATA_ADIMISSAO: string;
    CODIGO_FUNCAO: string;
    NOME_FUNCAO: string;
    CODIGO_EQUIPE: string;
    NOME_EQUIPE: string;
    CODIGO_DISCIPLINA: string;
    NOME_DISCIPLINA: string;
    PERFIL: string;
    CODIGO_OS: string;
    NOME_OS: string
}